.pre-commit-config.yaml

```diff
@@ -36,3 +37,12 @@ repos:
 #     hooks:
 #       - id: djlint-jinja
 #         types_or: ['html', 'jinja']
+
+  - repo: local
+    hooks:
+      - id: mypy
+        name: Run type checks
+        language: system
+        types: [python]
+        entry: mypy
```

Comment:
Can this run as `uv run mypy`, or does that break? 👀 

Reason being - otherwise you have to source the venv for mypy to be available, whereas `uv` should take care of that?

Reply:
Yeah that seems to work - done!